cloud/deploys/dev_azure/bin/destroy

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+source cloud/deploys/dev_azure/bin/lib.sh
+source cloud/azure/bin/lib/bastion.sh
+
+bastion::ssh_keygen "$HOME/.ssh/bastion"
+
+cloud/shared/bin/destroy
```

Comment:
yeah the image tag might get a lil lost here, i think you either need to source the config again or to have the user pass in the image_tag; it's unclear if it really matters like if you pass it in

Reply:
Hm, it seems to be running for me without needing to add a tag flag.  Is there something that you are seeing?  I don't think we really need a tag, right?

Reply:
(it runs for me without that flag because of the export TF_VAR_image_tag="${IMAGE_TAG}" in the other file)

Reply:
This one doesn't really need the image tag